evap/student/templates/student_vote.html

```diff
@@ -74,16 +75,18 @@
                 </div>
             </div>
         {% endif %}
+        {% if not for_rendering_in_modal %}
         <div class="well submit-area text-center">
             {% if preview %}
-                <a class="btn btn-default" onclick="preventAccidentalClosing = false" href="{{ request.META.HTTP_REFERER }}">{% trans "Back" %}</a>
+                <a class="btn btn-default" onclick="preventAccidentalClosing = false; window.open('', '_self', ''); window.close();">{% trans "Close" %}</a>
```

Comment:
can't you see it opens `''` in `_self`

no that's a leftover, the back button should stay as it is after the restructuring